app/controllers/staff/level_b_activity_uploads_controller.rb

```diff
@@ -30,7 +30,8 @@ def update
       importer = Activities::ImportFromCsv.new(
         uploader: current_user,
         partner_organisation: organisation,
-        report: nil
+        report: nil,
+        validation_steps: Activity::LEVEL_B_VALIDATION_STEPS
```

Comment:
Can we check this doesn't override the parent id uniqueness constraint here? We're assuming this only overrides the "not nil" requirement when skipping the `:identifier` step.

Reply:
Looks fine. I changed `valid_level_b_activities_upload.csv` to have `myui` as the `partner_organisation_identifier` for both activities and it came up with this error:

![image](https://user-images.githubusercontent.com/40244233/190469995-2b6066c8-3968-4d6c-90aa-6f2c5b34c4a6.png)